packages/react/src/components/ComposedModal/ComposedModal.jsx

```diff
@@ -222,7 +222,7 @@ class ComposedModal extends React.Component {
                 <Button
                   kind="secondary"
                   onClick={onClose}
-                  testID={`${testID}-modal-secondary-button`}
+                  testId={`${testID}-modal-secondary-button`}
```

Comment:
Did we miss the `testId` replacement for this file? I see we are passing on correctly to button but still have the prop of for `testID` for this component.

Reply:
these comments will be addressed in their own issues